main_logic/tts_client.py

```
@@ -1960,451 +2127,87 @@ def minimax_tts_worker(request_queue, response_queue, audio_api_key, voice_id, b
-                response_queue.put(("__audio__", accepted_speech_id, bytes(audio_chunk_buffer)))
-                audio_chunk_buffer.clear()
-
-        async def _wait_for_event(target_event: asyncio.Event, timeout_s: float) -> bool:
-            target_task = asyncio.create_task(target_event.wait())
-            disconnect_task = asyncio.create_task(disconnect_event.wait())
-            try:
-                done, pending = await asyncio.wait(
-                    {target_task, disconnect_task},
-                    timeout=timeout_s,
-                    return_when=asyncio.FIRST_COMPLETED,
+        # 连通性探测：HEAD 请求验证 API 可达 + 鉴权
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)